pages/visualizations/overview/commits_over_time.py

```diff
@@ -56,14 +57,15 @@
                                     dbc.RadioItems(
                                         id="commits-time-interval",
                                         options=[
-                                            {
-                                                "label": "Day",
-                                                "value": 86400000,
-                                            },  # days in milliseconds for ploty use
-                                            {
-                                                "label": "Week",
-                                                "value": 604800000,
-                                            },  # weeks in milliseconds for ploty use
+                                            # TODO:
```

Comment:
If you are going to use trend here can you change the label to trend? On my "TODOs" is to make the rest consistent on that side as well. If its trends lets call it that and same when we use daily